### PR TITLE
Security and parameters loading from super.json

### PR DESCRIPTION
**Pull request by freaz**
OneService was passing empty object `{}`, if something is passed, OneSDK uses it and ignores values in `super.json`

https://github.com/superfaceai/one-sdk-js/blob/69d86d18b8b6d2a1b1537da23cbd708c5409e0b9/src/core/profile-provider/profile-provider.ts#L129

### CODE REVIEW

**Review by jnv — APPROVED** [review]

File: src/one_sdk.spec.ts [hunk 1]
```diff
@@ -217,28 +217,32 @@ describe('one_sdk', () => {
     });
 
     describe('provider configurations', () => {
-      it('sets empty object if parameters are not set', async () => {
```
Comment (janhalama) [inline src/one_sdk.spec.ts:220]: question: Why did you remove the test for `if parameters are not set` case? The bug is not covered by the tests now.

**Review by janhalama — APPROVED** [review]
The fix looks good to me. I have one minor question regarding the unit tests.

File: src/one_sdk.spec.ts (hunk 1 above)
Comment (jnv) [reply src/one_sdk.spec.ts:220]: I was thinking the same but planned sending a separate PR to add the test case.

Comment (freaz) [reply src/one_sdk.spec.ts:220]: Ok I added it.
